### PR TITLE
fix some detail styles in the plugins

### PR DESCRIPTION
**Pull request by lichunn**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Release Notes

- **New Features**
	- Enhanced fullscreen functionality in the Monaco Editor.
	- Improved user interface for the Plugin Block List with new options for adding blocks.
	- Added search capabilities and improved layout in the Page Tree component.
	- Streamlined save settings interaction in the Save Toolbar.
	- Updated hover interactions for popover components to enhance user experience.

- **Bug Fixes**
	- Adjusted hover interactions for popover components across various areas.

- **Style**
	- Numerous style updates across components for improved visual consistency and responsiveness, including buttons, inputs, and tooltips.
	- Introduced new CSS variables for dynamic theming and hover effects.

- **Documentation**
	- Improved error handling and user feedback mechanisms in the internationalization management interface.

### CODE REVIEW

File: packages/design-core/assets/AI.svg [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 26.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (chilingling) [inline packages/design-core/assets/AI.svg]: svg  无用的注释删除一下

**Review by chilingling — DISMISSED** [review]

File: packages/plugins/page/src/PageGeneral.vue [hunk 2]
```diff
@@ -229,8 +229,6 @@ export default {
 
 <style lang="less" scoped>
 .general-config {
```
Comment (hexqi) [inline packages/plugins/page/src/PageGeneral.vue:231]: image
折叠面板下的间距看起来有点大，要看下是不是与设计稿相符，是否是折叠面板组件和内容区重复设置了间距

**Review by hexqi — COMMENTED** [review]
image
国际化面板操作按钮被遮挡

File: packages/plugins/datasource/src/DataSourceField.vue
Comment (hexqi) [inline packages/plugins/datasource/src/DataSourceField.vue:1]: image
image
数据源插件与资源管理插件的列表UI差异大

File: packages/plugins/datasource/src/DataSourceList.vue
Comment (hexqi) [inline packages/plugins/datasource/src/DataSourceList.vue:1]: image
数据源列表这里还有一个问题，鼠标划过之后，列表上还是会显示图标，没有隐藏

**Review by hexqi — APPROVED** [review]

**Pull request merged**